utils/ansible/ftest/templates/daos-launch_nlt.sh.j2

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+# shellcheck disable=SC1000-SC9999
```

Comment:
SC9999 doesn't appear to be valid and ideally we shouldn't be adding new code with suppressions but rather ensuring new code is correct/complaint to start with.

Reply:
Is it OK for you, if I fix this in a follow up PR ?

Reply:
No problem.